### ISSUE TITLE
compile error on fc21 64 bit

### ISSUE THREAD

**Issue by jakobkroeker** [jakobkroeker/test_M2#5]
workaround:
outcomment in in src/groebner/VectorArrayAPI.hpp

```
/*
#ifndef _4ti2_HAVE_MPZ_INT64_CONVERSION
template <>
inline
void
VectorArrayAPI::convert(const mpz_class& v1, _4ti2_int64_t& v2)
{
  std::cerr << "UNIMPLEMENTED: Need to convert from mpz to _4ti2_int64_t" << std::endl;
  exit(1);
}
#endif
*/

similar issue in src/zsolve/VectorArrayAPI.h

```

**Comment by jakobkroeker** [comment]
manually rerun HAVE_MPZ_INT64_CONVERSION test:

```
In file included from /usr/include/c++/4.9.2/cstdint:35:0,
                 from bla.cc:2:
/usr/include/c++/4.9.2/bits/c++0x_warning.h:32:2: Fehler: #error This file requires compiler and library support for the ISO C++ 2011 standard. This support is currently experimental, and must be enabled with the -std=c++11 or -std=gnu++11 compiler options.
```

**Comment by jakobkroeker** [comment]
fixed by dan (incorrect patch)
